src/mysql/populate.sql

```diff
@@ -27,12 +27,20 @@ INSERT INTO Notification (user_id, message, url, date_time)
 VALUES
   (4, 'You got an answer', 'questions/1', '2020-07-29 13:00:00.000000'),
   (6, 'You got an answer', 'questions/2', '2020-07-29 14:00:00.000000'),
-  (6, 'You got an answer', 'questions/2', '2020-07-29 15:00:00.000000'),
   (5, 'You got an answer', 'questions/3', '2020-07-29 16:00:00.000000'),
   (7, 'You got an answer', 'questions/4', '2020-07-29 17:00:00.000000'),
-  (7, 'You got an answer', 'questions/4', '2020-07-29 18:00:00.000000'),
-  (3, 'Somebody commented your answer', 'questions/2', '2020-07-29 19:00:00.000000'),
-  (3, 'Somebody commented your answer', 'questions/2', '2020-07-29 20:00:00.000000');
+  (3, 'Somebody commented your answer', 'questions/2', '2020-07-29 19:00:00.000000');
+
+INSERT INTO UserNofication (user_id, notification_id)
```

Comment:
it says UserNofication instead of UserNotification

Reply:
Fixed!